gwvet/tabs.py

```diff
@@ -190,6 +192,14 @@ def from_ini(cls, config, section, **kwargs):
                                      combine)
                 else:
                     kwargs['intersection'] = False
+        # get glitchgram plotting arguments
+        kwargs['glitchgramargs'] = {key.split('glitchgram-')[1]: val for
+                                    key, val in config.items(section) if
```

Comment:
@tjma12, for pedantry's sake, can you wrap the `key, val` dummy variables as an explicit tuple?

```suggestion
                                    (key, val) in config.items(section) if
```